editor/src/messages/portfolio/document/node_graph/graph_operation_message_handler.rs

```diff
@@ -510,17 +510,62 @@ impl<'a> ModifyInputsContext<'a> {
 		});
 	}
 
-	fn delete_layer(&mut self, id: NodeId, selected_nodes: &mut SelectedNodes) {
+	fn delete_layer(&mut self, id: NodeId, selected_nodes: &mut SelectedNodes, mut is_artboard_layer: bool) {
```

Comment:
The function signature shouldn't have this be `mut`, since we don't want the caller to care about the implementation detail. You can redeclare the variable to be mut in the body of the function.